vision_evaluation/evaluators.py

```diff
@@ -130,13 +133,18 @@ def calculate_score(self, average='macro'):
         if self.all_predictions.size == 0:
             return 0.0
 
-        tar_mat = _targets_to_mat(self.all_targets, self.all_predictions.shape[1])
+        tar_mat = _targets_to_mat(self.all_targets,
+                                  self.all_predictions.shape[1])
         assert tar_mat.size == self.all_predictions.size
         result = 0.0
         if tar_mat.size > 0:
-            non_empty_idx = np.where(np.invert(np.all(tar_mat == 0, axis=0)))[0]
+            non_empty_idx = np.where(np.invert(np.all(tar_mat == 0,
+                                                      axis=0)))[0]
             if non_empty_idx.size != 0:
-                result = self._calculate(tar_mat[:, non_empty_idx], self.all_predictions[:, non_empty_idx], average=average)
+                result = self._calculate(tar_mat[:, non_empty_idx],
```

Comment:
this seems very hard to read. Please help revert

Reply:
updated